github-actions/integration-tests/action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Integration Tests'
+description: 'Runs the integration tests for the given app repo through AWS CodeBuild'
+inputs:
+  AWS_ROLE_ARN:
+    description: 'Role to assume to run the CodeBuild project'
+    required: true
+  AWS_REGION:
+    description: 'Region of the CodeBuild project'
+    default: 'us-west-2'
+    required: false
+  CODE_BUILD_PROJECT:
+    description: 'Name of the codebuild project'
+    default: 'soar-connector-integration-tests'
+    required: false
+runs:
+  using: 'composite'
+  steps:
+    - name: 'Checkout App Repo'
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.ref }}
+    - name: 'Configure AWS Credentials'
+      uses: aws-actions/configure-aws-credentials@v1.6.1
```

Comment:
so from skimming the docs on this, this action sets the credentials based on expected env variable names? hence, why we don't need to explicitly specify them in the env here?

Reply:
neat! thanks for the link